README.md

```diff
@@ -22,10 +19,28 @@ Proposal to define an XML-based logging format for outputs from EDA tools and lo
 * State Coverage
 * Transition Coverage
 
-## Functional Coverage
+### Functional Coverage
 
 * Coverage Model
   * Dimension
   * Bin
     * Item
     * Range
+
+### Synthesis
+
+* Area/resources
+* Timing
```

Comment:
* output messages in structured format
  * categorize: message IDs
  * categorize: errors, warnings, infos

Reply:
Now explained in the summary.